edgraph/server.go

```diff
@@ -2005,9 +2356,9 @@ func validateAndConvertFacets(nquads []*api.NQuad) error {
 	return nil
 }
 
-// validateForGraphql validate nquads for graphql
-func validateForGraphql(nq *api.NQuad, isGraphql bool) error {
-	// Check whether the incoming predicate is graphql reserved predicate or not.
+// validateForOtherReserved validate nquads for other reserved predicates
```

Comment:
What are the others? Can you explain a bit here

Reply:
Defined here https://github.com/hypermodeinc/dgraph/blob/main/x/keys.go#L663

Reply:
I meant, can you add a comment there for future?